flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -432,10 +452,14 @@ func (c *ElasticSearchStorage) SearchFlows(fsq filters.SearchQuery) (*flow.FlowS
 
 // Start the Database client
 func (c *ElasticSearchStorage) Start() {
-	go c.client.Start([]map[string][]byte{
+	entriesLimit := config.GetInt("analyzer.storage.index_entries_limit")
+	ageLimit := config.GetInt("analyzer.storage.index_age_limit")
+	indicesLimit := config.GetInt("analyzer.storage.indices_to_keep")
+	go c.client.Start("flows", []map[string][]byte{
```

Comment:
again - better is storage module is not using configuration from the analyzer section